.github/workflows/tests-linux.yml

```diff
@@ -0,0 +1,58 @@
+name: Linux CI
```

Comment:
I don't think we're ready to be adding Linux-only CI to Homebrew/homebrew-core yet.